pkg/controller/acicontainersoperator.go

```diff
@@ -639,6 +642,11 @@ func (c *Controller) handleOperatorCreate(obj interface{}) bool {
 
 	if acicontainersoperator.Spec.Config == "" {
 		log.Info("acicnioperator CR config is Nil...Exiting")
+		acicontainersoperator.Status.Status = false
+		_, er := c.Operator_Clientset.AciV1alpha1().AciContainersOperators(os.Getenv("SYSTEM_NAMESPACE")).Update(acicontainersoperator)
```

Comment:
Did you verify? that Status is getting updated.  there is call yo update only status UpdateStatus.

Reply:
yes verified on snatjordan cluster , with CR as nil and status updated as false
apiVersion: aci.ctrl/v1alpha1
kind: AciContainersOperator
metadata:
  creationTimestamp: "2020-05-29T22:34:24Z"
  generation: 2
  name: acicnioperator
  namespace: aci-containers-system
  resourceVersion: "929236"
  selfLink: /apis/aci.ctrl/v1alpha1/namespaces/aci-containers-system/acicontainersoperators/acicnioperator
  uid: 9b3437c5-1d87-4e5c-9aff-22d061ead9e0
spec:
  config: ""
  flavor: ""
status:
  Successful: false